website/utils/media/services.py

```diff
@@ -64,24 +53,16 @@ def get_thumbnail_url(file, size, fit=True):
         name = file.name
 
     is_public = isinstance(storage, get_storage_class(settings.PUBLIC_FILE_STORAGE))
-    size_fit = f"{size}_{int(fit)}"
 
     if name.endswith(".svg") and is_public:
         return storage.url(name)
 
-    sig_info = {
-        "size": size,
-        "fit": int(fit),
-        "name": name,
-        "thumb_path": f"thumbnails/{size_fit}/{name}",
-        "serve_path": f"thumbnails/{size_fit}/{name}",
-        "storage": f"{storage.__class__.__module__}.{storage.__class__.__name__}",
-    }
+    if isinstance(file, ThumbnailedImageFile):
+        if size == "small":
+            return get_media_url(file.thumbnails.small)
+        if size == "medium":
+            return get_media_url(file.thumbnails.medium)
```

Comment:
There are other sizes defined in settings, why aren't those in here too?

Reply:
Update: the others are used (in ThumbnailSerializers in the API), so I guess they should be used.